play-json/shared/src/main/scala/play/api/libs/json/JsLookup.scala

```diff
@@ -173,13 +156,37 @@ object JsLookupResult {
 /**
  * Wrapper for JsValue to represent an existing Json value.
  */
-case class JsDefined(value: JsValue) extends AnyVal with JsLookupResult
+case class JsDefined(value: JsValue) extends AnyVal with JsLookupResult {
+
+  override def toOption: Option[JsValue] = Some(value)
```

Comment:
`override` modifier is not required (and doesn't make sense) as there is no implementation in the parent type overridden there